dev/com.ibm.ws.request.timing/src/com/ibm/ws/request/timing/probeExtensionImpl/SlowRequestProbeExtension.java

```diff
@@ -75,6 +78,16 @@ public void processExitEvent(Event event, RequestContext requestContext) {
 			//Clean up activity for this request will be performed in the corresponding slow request timer task.
 
 			/** Instead of this should we just return false from invokeForEventExit? */
+		} else {
```

Comment:
I think this code is in the wrong half of the if block.  It looks like the else block will only run when the requestTiming feature is stopping.